python/lsst/ip/isr/isrTask.py

```diff
@@ -2127,6 +2141,43 @@ def maskEdges(self, exposure, numEdgePixels=0, maskPlane="SUSPECT"):
                 maskBitMask
             )
 
+    def maskGrowTophat(self, exposure, boxRadius=0, boxThreshold=0.25, maskPlane="BAD", maskLabel="BAD"):
```

Comment:
Delete the definition of  `maskGrowTophat`. It'll bitrot without a unit test or use anywhere